bind/gencffi.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2017 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bind
+
+import (
+	"go/token"
+)
+
+const (
+	builderPreamble = `import os
+
+from sys import argv
+from cffi import FFI
+
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+ffibuilder = FFI()
+ffibuilder.set_source(
+    'gopy.%[1]s',
+    None,
+    extra_objects=["gopy/_%[1]s.so"],
```

Comment:
The first problem if we do not create an extra directory.
If we generate with `simple.go`, CFFI will generate `simple.py` also.(This flow is uncontrollable.) 
This file name is duplicated. Because we also need to generate `simple.py` also for user importing files.
(I think that user wants to import our generated files as  not `import gosimple` but 'import simple`.)
So that's why I create the extra directory as `gopy` and then put it into. And then we could write wrapper python file the same name as `simple.py'.

The second problem is that It will be little dirty because there are 4 output files. So if we put most of them into `gopy` directory. the user can easily remove them.